### ISSUE TITLE
module 'distutils.config' has no attribute 'ConfigParser'

### ISSUE THREAD

**Issue by gijzelaerr** [aplpy/aplpy#304]
Hi! i can't install aplpy in a clean python3 virtualenv:

```
$  virtualenv -p python3 aplpy 
Running virtualenv with interpreter /usr/bin/python3
Using base prefix '/usr'
New python executable in /home/gijs/Tmp/aplpy/bin/python3
Also creating executable in /home/gijs/Tmp/aplpy/bin/python
Installing setuptools, pip, wheel...done.

$  source aplpy/bin/activate

$  pip install aplpy
Collecting aplpy
  Using cached APLpy-1.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-mcvxzzef/aplpy/setup.py", line 25, in <module>
        conf = config.ConfigParser()
    AttributeError: module 'distutils.config' has no attribute 'ConfigParser'

$  aplpy/bin/python -V
Python 3.5.2

$  aplpy/bin/pip -V
pip 8.1.2 from /home/gijs/Tmp/aplpy/lib/python3.5/site-packages (python 3.5)
```

**Comment by evertrol** [comment]
Fwiw: this was fixed in [commit fddb0a6](https://github.com/aplpy/aplpy/commit/fddb0a60551707a0f5eeb3b66924d4d295ade8b2). 
Installation from the current master will do it: 
```
python3.5 -m pip install git+https://github.com/aplpy/aplpy.git
```

**Comment by astrofrog** [comment]
Closing this since this is now fixed, and I'll be releasing a new version very shortly
